### ISSUE TITLE
Move to a single activity app

### ISSUE THREAD

**Issue by wb9688** [TeamNewPipe/NewPipe#3477]
Currently we have some activities besides `MainActivity`, such as `AboutActivity` or `SettingsActivity`, while they should be fragments imho.

**Comment by opusforlife2** [comment]
Would solve #4135?

**Comment by opusforlife2** [comment]
Downloads as well.

**Comment by wb9688** [comment]
@opusforlife2: Yes, but not worth it to move that into the single activity if we want to make a new UI soon-ish.

**Comment by opusforlife2** [comment]
There's a new UI in the works? O_O
